samples/csharp_dotnetcore/06.using-cards/Bots/RichCardsBot.cs

```diff
@@ -26,7 +31,10 @@ protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersA
                 // To learn more about Adaptive Cards, see https://aka.ms/msbot-adaptivecards for more details.
                 if (member.Id != turnContext.Activity.Recipient.Id)
                 {
-                    var reply = MessageFactory.Text($"Welcome to CardBot {member.Name}. This bot will show you different types of Rich Cards. Please type anything to get started.");
+                    var reply = MessageFactory.Text("Welcome to CardBot."
+                        + " This bot will show you different types of Rich Cards."
+                        + " Please type anything to get started.");
+
                     await turnContext.SendActivityAsync(reply, cancellationToken);
                 }
             }
```

Comment:
I'm a little ambivalent about this change. I think its fine to leave it under the old name.
However, that does kind of imply its more generic than it is. So I'm OK with renaming this.
And again NO xml comments.